routes/web.php

```diff
@@ -14,14 +14,15 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+
 
 Route::get('/register', [RegisterController::class, 'register']);
 Route::get('/login', [RegisterController::class, 'login']);
 
 Route::post('/register-user',[RegisterController::class,'registerUser'])->name('register-user');
 Route::post('login-user', [RegisterController::class, 'loginUser'])->name('login-user');
 
-Route::get('/dashboard',[RegisterController::class,'dashboard'])->middleware('isLoggedIn');
\ No newline at end of file
+Route::get('/home',[RegisterController::class,'dashboard'])->middleware('isLoggedIn');
```

Comment:
```suggestion
Route::get('/home',[RegisterController::class,'dashboard'])->middleware('isLoggedIn')->name('home');
```
Always name routes and use name for clean codes.

Similar changes on all routes.

Reply:
PR updated as per reviews.